docs/gpa.md

```diff
@@ -475,47 +475,56 @@ This requires the [Participation Fees](https://github.com/open-contracting-exten
         <td>a description of the goods or services procured;</td>
         <td markdown=1>
 
-* 
+* Follow the guidance for [XIII:2](#XIII:2).
```

Comment:
I agree, like in your issue description, that it's hard to follow the reference. For this article, let's instead repeat the specific guidance, and mention that award notices are also discussed in XIII:2.

Reply:
OK.
I forgot to write guidance for "lowest offers taken into account in   the award of the contract".

Reply:
Done both